image/provider.tf

```diff
@@ -0,0 +1,3 @@
+provider "libvirt" {
+  uri = "qemu:///system"
+}
```

Comment:
I believe it's the implicit default. You don't need to put it.

Reply:
I had an error that the URI was missing, that's why I added it.